src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/AprilTagCamera.java

```diff
@@ -67,21 +64,15 @@ public double getLatestResultTimestampSeconds() {
         return inputs.latestResultTimestampSeconds;
     }
 
-    public void setGyroHeadingToSolvePNPHeading() {
-        final Translation2d currentTranslation = RobotContainer.POSE_ESTIMATOR.getCurrentPose().getTranslation();
-        final Rotation2d solvePNPHeading = inputs.cameraSolvePNPPose.getRotation().toRotation2d();
-        RobotContainer.POSE_ESTIMATOR.resetPose(new Pose2d(currentTranslation, solvePNPHeading));
-    }
```

Comment:
implement this somewhere, probably pose estimator (obviously not as simple as copy paste, you should reset to the camera with the smallest distance from best tag)